extensions/extensions-helm/aissemble-mlflow-chart/values.yaml

```diff
@@ -1,10 +1,7 @@
 mlflow:
   image:
-    # NB: will soon be updated to public mlflow image - but waiting to upgrade across the application, so point to the
-    # 2.3.1 image that was part of aissemble 1.7.0 in the meantime. TODO in https://github.com/boozallen/aissemble/issues/226
-    registry: ghcr.io
-    repository: boozallen/aissemble-mlflow
-    tag: 1.7.0
+    registry: docker.io
+    repository: bitnami/mlflow
```

Comment:
S: If these were removed from the README table because they are the defaults, they should be removed from here. Looks like we should remove the entire `image` block

Reply:
Good catch. Removed them.